In order to avoid a future red travis result, Could you include fix lint?

``` bash
/home/travis/build/Vauxoo/addons-vauxoo/default_warehouse_from_sale_team/__openerp__.py:15:9: E265 block comment should start with '# '
/home/travis/build/Vauxoo/addons-vauxoo/default_warehouse_from_sale_team/__openerp__.py:16:9: E265 block comment should start with '# '
/home/travis/build/Vauxoo/addons-vauxoo/default_warehouse_from_sale_team/models/sales_team.py:6:1: E302 expected 2 blank lines, found 1
************* Module default_warehouse_from_sale_team.models.sales_team
default_warehouse_from_sale_team/models/sales_team.py:97: [W0621(redefined-outer-name), WarehouseDefault.read] Redefining name 'fields' from outer scope (line 3)
default_warehouse_from_sale_team/models/sales_team.py:110: [W0621(redefined-outer-name), WarehouseDefault.read] Redefining name 'fields' from outer scope (line 3)
default_warehouse_from_sale_team/models/sales_team.py:110: [E0102(function-redefined), WarehouseDefault.read] method already defined line 97
************* Module default_warehouse_from_sale_team
default_warehouse_from_sale_team/README.rst:6: [E7901(rst-syntax-error), ]  Enumerated list start value not ordinal-1: "3" (ordinal 3)
```

--- a/default_warehouse_from_sale_team/README.rst
+++ b/default_warehouse_from_sale_team/README.rst
@@ -52,7 +52,7 @@ with the user sale teams default_warehouse. To accomplish this I:
 
 To add more models use it simple do this:
 
-1. inherit the class that you want to set the field warehouse_id::
+1. Inherit the class that you want to set the field warehouse_id::
 
     class SomeClass(models.Model):
         _name = 'some.class'
@@ -61,8 +61,8 @@ To add more models use it simple do this:
         'this object will belong to')
 
 2. Create two ir.rule to filter stock.picking and stock.picking.type taking
-into account the current user warehouses. When a user is part of warehouse
-teams will be able to access only the records related to that warehouses::
+   into account the current user warehouses. When a user is part of warehouse
+   teams will be able to access only the records related to that warehouses::
 
     <record id="rule_group_model" model="ir.rule">
         <field name="name">Limited access to model (filtered by sales teams)</field>
@@ -77,12 +77,12 @@ teams will be able to access only the records related to that warehouses::
         <field name="domain_force">[(1, '=', 1)]</field>
     </record>
 
-3. Don't forget depends of this module adding it to the list into `__openerp__.py`
+3. Don't forget depends of this module adding it to the list into `__openerp__.py`::
 
-The default value from this field will be the warehouse setted in the section
-If the user is not related to a sales team or not warehouse setted on the
-section the default warehouse will be set using the default behavior of the
-system which is assign the main warehouse.
+    The default value from this field will be the warehouse setted in the section.
+    If the user is not related to a sales team or not warehouse setted on the
+    section the default warehouse will be set using the default behavior of the
+    system which is assign the main warehouse.
 
 Bug Tracker
 ===========

--- a/default_warehouse_from_sale_team/__openerp__.py
+++ b/default_warehouse_from_sale_team/__openerp__.py
@@ -12,8 +12,8 @@
     'version': '8.0.2.0.0',
     'depends': [
         'sale_stock',
-        # TODO: Remove "delivery" if the following issue is fixed
-        # https://github.com/odoo/odoo/pull/13458
+        #  TODO: Remove "delivery" if the following issue is fixed
+        #  https://github.com/odoo/odoo/pull/13458
         'delivery',
         'sales_team',
         'base_action_rule',

--- a/default_warehouse_from_sale_team/models/sales_team.py
+++ b/default_warehouse_from_sale_team/models/sales_team.py
@@ -3,6 +3,7 @@
 from openerp import api, fields, models
 from openerp import SUPERUSER_ID
 
+
 class InheritedCrmSaseSection(models.Model):
 
     _inherit = "crm.case.section"
@@ -94,7 +95,7 @@ class WarehouseDefault(models.Model):
         return defaults
 
     @api.v7
-    def read(self, cr, user, ids, fields=None, context=None,
+    def read(self, cr, user, ids, fields_list=None, context=None,
              load='_classic_read'):
         """This method is overwrite because we need to propagate SUPERUSER_ID
         when picking are chained in another warehouse without access to read"""
@@ -104,14 +105,14 @@ class WarehouseDefault(models.Model):
             # we need to change to SUPERUSER_ID to allow access to read
             user = SUPERUSER_ID
         return super(WarehouseDefault, self).read(
-            cr, user, ids, fields=fields, context=context, load=load)
+            cr, user, ids, fields=fields_list, context=context, load=load)
 
     @api.v8
-    def read(self, fields=None, load='_classic_read'):
+    def read(self, fields_list=None, load='_classic_read'):
         """This method is overwrite because we need to propagate SUPERUSER_ID
         when picking are chained in another warehouse without access to read"""
         if self.env.user.has_group('default_warehouse_from_sale_team.'
                                    'group_limited_default_warehouse_sp'):
             # we need to change to SUPERUSER_ID to allow access to read
             self = self.sudo()
-        return super(WarehouseDefault, self).read(fields, load)
+        return super(WarehouseDefault, self).read(fields_list, load)

